docs/jwst/pub_deprecation.rst

```diff
@@ -0,0 +1,71 @@
+.. _pub-deprecation:
+
+CRDS PUB Server Freeze and Deprecation
+======================================
+
+Why and When
+------------
+
+As of November 10, 2022, all observers should use the standard CRDS OPS server
+for JWST calibration reference files:
+
+`https://jwst-crds.stsci.edu <https://jwst-crds.stsci.edu>`_
+
+The PUB server:
+
+`https://jwst-crds-pub.stsci.edu <https://jwst-crds-pub.stsci.edu>`_
+
+was set up in anticipation of rapid reference file updates during commissioning
+and Cycle 1. However, due to the trouble-free commissioning process,
+the smooth transition to science operations, and the subsequent
+confusion that has resulted from having two servers, it has been
+decided that the PUB server is no longer needed and will be
+decommissioned. To make this transition as smooth as possible, this
+update will take place in stages.
+
+On November 10, 2022, all observers should begin to transition to using only the CRDS
+OPS server, `https://jwst-crds.stsci.edu <https://jwst-crds.stsci.edu>`_. See
+the :ref:`software documentation <pub-deprecation-transition>` for
+instructions about how to configure CRDS.
+
+In mid-November, the PUB server will be frozen and a notification will be sent via the
+reference file mailing lists and the JWST Observer News mailing list.
+
+On December 2nd, access to the PUB server will no longer be available externally.
+The frozen PUB database will be maintained internally for 3 months. On March 1, the PUB server will
+be fully decommissioned and the institute will
+retain an internal archive of the maps and calibration reference files.
+Observers who wish to use historical files from the PUB server in the future will need to file a
+JWST Pipeline help desk ticket to access the information.
+
+Part of the decommissioning process will include establishing guidance for how best to
+maintain reproducibility for new papers and for already-published papers that used the PUB server. This information
+will be included in the new JDox page:
+
+`How to Cite Your Data Reduction and Reference Files <https://jwst-docs.stsci.edu/display/JDOX/.How+to+Cite+Your+Data+Reduction+and+Reference+Files+v1.2>`_
```

Comment:
This link url doesn't work. Looks like it needs to be
https://jwst-docs.stsci.edu/display/JDOX/.How+to+Cite+JWST+Data+Reductions+and+Reference+Files+v1.2

Reply:
Of course given that this jdox page isn't actually published yet (as far as I know), the url may change again once it does get published?  I guess we just wait and see.

Reply:
@rizeladiaz says the correct public link will be:
https://jwst-docs.stsci.edu/jwst-science-calibration-pipeline-overview/jwst-data-calibration-reference-files